app/controllers/static_dashboards_controller.rb

```diff
@@ -0,0 +1,44 @@
+require 'server_transformer'
+class StaticDashboardsController < ApplicationController
+  protect_from_forgery with: :exception, only: :update
+  before_action :set_dashboard, only: :show
+  before_action :set_directories, only: :show
+
+  def show
+    @servers = Server.order("lower(name)")
+    render 'dashboards/show'
+  end
+
+  def create
+    @dashboard = Dashboard.create_static(dashboard_params)
+    if @dashboard.save
+      payload = {
+        url: "/static/" + SlugMaker.slug("#{@dashboard.id} #{@dashboard.slug}")
+      }
+      render json: payload, status: :created
+    else
+      render json: @dashboard.errors, status: :unprocessable_entity
+    end
+  end
+
+  private
+  # Use callbacks to share common setup or constraints between actions.
+  def set_dashboard
+    @dashboard = Dashboard.find(params[:id])
+  end
+
+  def set_directories
+    @directories = Directory.all
+  end
+
+  # Never trust parameters from the scary internet, only allow the white list through.
```

Comment:
Seems like duplicating some complex code. I'm wondering whether to have just one controller for both types of dashboard and just send the "static" attribute as part of the normal POST data instead of differentiating it in the URL.